meson.build

```diff
@@ -2344,13 +2344,13 @@ if git.found()
                  ':/*.[ch]'])
         all_files = files(all_files.stdout().split())
 
-        run_target(
+        custom_target(
                 'tags',
-                input : all_files,
```

Comment:
FYI: Removing the `input` is intentional. `custom_target` (unlike `run_target`) automatically adds `files` appearing in `command` as implicit deps in `ninja.build`.